modules/olm-installing-po-after.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-po.adoc
+
+:_content-type: PROCEDURE
+[id="olm-installing-po-after_{context}"]
+= Installing platform Operators after cluster creation
+
+As a cluster administrator, you can install platform Operators after cluster creation on clusters that have enabled the `TechPreviewNoUpgrades` feature set by using the cluster-wide `PlatformOperator` API.
```

Comment:
Maybe we can add the below command for users.
Note that, once enabled, tech preview features may not be disabled.
```console
MacBook-Pro:~ jianzhang$ oc patch featuregate cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' --type=merge
featuregate.config.openshift.io/cluster patched
MacBook-Pro:~ jianzhang$ oc get pods -n openshift-platform-operators
NAME                                                     READY   STATUS    RESTARTS   AGE
platform-operators-controller-manager-77d5d88cc6-w48jj   2/2     Running   0          2m4s
platform-operators-rukpak-core-c5978759-pxvq9            2/2     Running   0          2m4s
platform-operators-rukpak-webhooks-758764ff88-855d6      1/1     Running   0          2m4s
platform-operators-rukpak-webhooks-758764ff88-wh42m      1/1     Running   0          2m4s

```